src/main/java/minicraft/screen/entry/StringEntry.java

```diff
@@ -21,29 +22,29 @@ public static StringEntry[] useLines(int color, String... lines) {
 		StringEntry[] entries = new StringEntry[lines.length];
 		for (int i = 0; i < lines.length; i++)
 			entries[i] = new StringEntry(lines[i], color);
-		
+
 		return entries;
 	}
-	
+
 	public StringEntry(String text) {
 		this(text, DEFAULT_COLOR);
 	}
 	public StringEntry(String text, int color) {
 		setSelectable(false);
-		this.text = text;
+		this.text = Localization.getLocalized(text);
```

Comment:
Change this to only localize if you explicitly tell it to. Take a look at how it is done in SelectEntry, and replicate that.

This also gives me an idea where we completely remove text, and replace it with an abstract string in the ListEntry class that represents the id of the entry, or maybe a total overhaul of the gui system, but that is for later.

Reply:
That would be in #313